### PR TITLE
Upgrade to CircleCi 2.0

### PR DESCRIPTION
**Pull request by f-moya**

### CODE REVIEW

**Comment by f-moya** [comment]
I think circleci it's not active for this repository. I tried to enable it but need permissons :grimacing:
Screen Shot 2020-05-14 at 12 04 26 PM

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 2
```
Comment (filipesperandio) [inline .circleci/config.yml:1]: Thursday I tried to set it up, got into a weird state. Surprised it worked after that.

There is some weirdness now with the new Pipelines setup, that I also enabled with the new UI stuff, saw something similar for the browser repo while I wasn't seeing any builds, moving it into `2.1` seemed to help. I think we should set 2.1 as the default going forward for all repos.

**Comment by f-moya** [comment]
I'm pretty sure this repo will also demand to add a `deploy-key` on CircleCI's regarding project configuration, in order for CircleCI to clone the repo.

**Comment by f-moya** [comment]
Not sure of what changed, but seems like it's working now 🥇

File: .circleci/config.yml (hunk 1 above)
Comment (f-moya) [reply .circleci/config.yml:1]: Ok - Will upgrade to 2.1 then.  (Will check circle docs a little bit first)

Comment (filipesperandio) [reply .circleci/config.yml:1]: Sure.
Haven't spent much on it, but I understand 2.1 is backward compatible (as it should) and enable so extra features coming with this new _Pipeline_ thing they are introducing on top of the _Workflow_ and goes along with the new UI.
In terms of our own config/steps, I don't think anything has to change.

Comment (filipesperandio) [reply .circleci/config.yml:1]: You could have kept the PR 🤷

Comment (f-moya) [reply .circleci/config.yml:1]: haha, yes I know, you know I renamed the brach and when I realised the PR was closed

Comment (filipesperandio) [reply .circleci/config.yml:1]: gotcha...

**Pull request closed without merging**